<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>432</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="dMain">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <widget class="QFrame" name="frmMain">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>128</width>
         <height>256</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="frmProperty">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabList">
       <attribute name="title">
        <string>对象列表</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QListWidget" name="lstObjects"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabProperty">
       <attribute name="title">
        <string>属性</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <property name="verticalSpacing">
         <number>1</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="lblShow1">
          <property name="text">
           <string>元件属性</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblShow2">
          <property name="text">
           <string>名称（用于记忆）：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="txtName"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lblShow4">
          <property name="text">
           <string>CSS选择器：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="txtID"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="lblShow5">
          <property name="text">
           <string>数据：</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="txtData"/>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="lblShow3">
          <property name="text">
           <string>宽度</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLabel" name="lblShow6">
          <property name="text">
           <string>高度</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="lblShow8">
          <property name="text">
           <string>Left</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="lblShow7">
          <property name="text">
           <string>Top</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QPushButton" name="btnClose">
          <property name="text">
           <string>删除元件</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QPushButton" name="btnOk">
          <property name="text">
           <string>保存属性</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QSpinBox" name="txtLeft">
          <property name="minimum">
           <number>-65535</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QSpinBox" name="txtTop">
          <property name="minimum">
           <number>-65535</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QSpinBox" name="txtWidth">
          <property name="minimum">
           <number>-65535</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QSpinBox" name="txtHeight">
          <property name="minimum">
           <number>-65535</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>导出</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPlainTextEdit" name="txtExportHTML"/>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="txtExport"/>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="txtExportCSS"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mnuMain">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>896</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="btnFileOpen"/>
    <addaction name="btnFileSave"/>
    <addaction name="btnFileSaveAs"/>
    <addaction name="separator"/>
    <addaction name="btnExit"/>
   </widget>
   <widget class="QMenu" name="mnuInsert">
    <property name="title">
     <string>插入</string>
    </property>
    <addaction name="btnInsertImage"/>
    <addaction name="btnInsertLogic"/>
   </widget>
   <widget class="QMenu" name="mnuEdit">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="btnPropertyWindow"/>
    <addaction name="separator"/>
    <addaction name="btnExportPNG"/>
    <addaction name="btnExportCSS"/>
    <addaction name="btnExportSelf"/>
   </widget>
   <widget class="QMenu" name="mnuObjects">
    <property name="title">
     <string>元件</string>
    </property>
    <addaction name="btnOpenSubWin"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="mnuFile"/>
   <addaction name="mnuInsert"/>
   <addaction name="mnuEdit"/>
   <addaction name="mnuObjects"/>
  </widget>
  <widget class="QStatusBar" name="lblStatus"/>
  <action name="btnFileOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="action" stdset="0">
    <number>0</number>
   </property>
  </action>
  <action name="btnFileSave">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="action" stdset="0">
    <number>1</number>
   </property>
  </action>
  <action name="btnFileSaveAs">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="action" stdset="0">
    <number>2</number>
   </property>
  </action>
  <action name="btnExit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="action" stdset="0">
    <number>3</number>
   </property>
  </action>
  <action name="btnInsertImage">
   <property name="icon">
    <iconset theme="insert-image">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>图片</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="action" stdset="0">
    <number>100</number>
   </property>
  </action>
  <action name="btnInsertLogic">
   <property name="icon">
    <iconset theme="draw-rectangle">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>逻辑区域</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="action" stdset="0">
    <number>101</number>
   </property>
  </action>
  <action name="btnPropertyWindow">
   <property name="icon">
    <iconset theme="transform-crop-and-resize">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>图像设定</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="action" stdset="0">
    <number>200</number>
   </property>
  </action>
  <action name="btnOpenSubWin">
   <property name="icon">
    <iconset theme="preferences-system-windows-actions">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>属性窗口</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="action" stdset="0">
    <number>300</number>
   </property>
  </action>
  <action name="btnExportCSS">
   <property name="icon">
    <iconset theme="text-css">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>导出独立CSS</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="action" stdset="0">
    <number>202</number>
   </property>
  </action>
  <action name="btnExportPNG">
   <property name="icon">
    <iconset theme="image-x-generic" resource="../Resources/res.qrc">
     <normaloff>:/new-image.png</normaloff>:/new-image.png</iconset>
   </property>
   <property name="text">
    <string>导出PNG</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="action" stdset="0">
    <number>201</number>
   </property>
  </action>
  <action name="btnExportSelf">
   <property name="icon">
    <iconset theme="text-css">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>导出CSS到当前文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="action" stdset="0">
    <number>203</number>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Resources/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>mnuMain</sender>
   <signal>triggered(QAction*)</signal>
   <receiver>MainWindow</receiver>
   <slot>onMenuClicked(QAction*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onMenuClicked(QAction*)</slot>
 </slots>
</ui>
